### PR TITLE
Sending test properties in TestItemFinishedEventArgs

### PR DESCRIPTION
**Pull request by undron**
There is quit a lot of properties in test (see PropertyAttribute in NUnit), would be good to handle them in customization.

### CODE REVIEW

**Comment by undron** [comment]
Now sending original report string from NUnit

File: ReportPortal.NUnitExtension/ReportPortalListener.Suite.cs [hunk 1]
```diff
@@ -120,7 +120,7 @@ private void FinishSuite(XmlDocument xmlDoc)
                             finishSuiteRequest.Description = description.Attributes["value"].Value;
                         }
 
-                        var eventArg = new TestItemFinishedEventArgs(Bridge.Service, finishSuiteRequest, _flowItems[id].Reporter);
+                        var eventArg = new TestItemFinishedEventArgs(Bridge.Service, finishSuiteRequest, _flowItems[id].Reporter, xmlDoc.InnerXml);
```
Comment (nvborisenko) [inline ReportPortal.NUnitExtension/ReportPortalListener.Suite.cs]: `OuterXml` covers more than `InnerXml`

Comment (undron) [reply ReportPortal.NUnitExtension/ReportPortalListener.Suite.cs]: Done

Comment (undron) [reply ReportPortal.NUnitExtension/ReportPortalListener.Suite.cs]: But IMHO for XmlDocument this should be the same

**Pull request merged**